lib/active_job/jobs_relation.rb

```diff
@@ -23,9 +23,9 @@ class ActiveJob::JobsRelation
   include Enumerable
 
   STATUSES = %i[ pending failed in_progress blocked scheduled finished ]
-  FILTERS = %i[ queue_name job_class_name ]
+  FILTERS = %i[ queue_name job_class_name finished_at_start finished_at_end ]
```

Comment:
I am not sure I we should add this to filters given that the filtering is done natively for solid_queue and i'ts not supported for resque (there isn't a `finished` status)

```suggestion
  FILTERS = %i[ queue_name job_class_name ]
```

Reply:
@rosa what are your thoughts on this?

Reply:
This would also allow to not show `finished_at` related filters for the status where they don't make sense. They only make sense for finished jobs.

Reply:
yepp, that makes sense, I'll give it a try

Reply:
hi @rosa, I was thinking about doing this mini refactor in another pr, considering that the possible breaking behaviour (trying to filter a jobs_relation by finished_at for an adapter that does not support it) should not happen.

I think it's a clear enhacement for codebase, but I would rather do it in a separate issue/pr